src/components/Users/UserListAndCard.tsx

```diff
@@ -81,9 +88,21 @@ const getNameAndStatusCard = (
       <div className="flex flex-row justify-between gap-x-3">
         <div className="flex flex-col">
           <div className="flex items-center gap-x-3">
-            <h1 id={`name-${user.username}`} className="text-base font-bold">
-              {formatName(user)}
-            </h1>
+            <TooltipProvider delayDuration={200}>
+              <Tooltip>
+                <TooltipTrigger asChild>
+                  <h1
+                    id={`name-${user.username}`}
+                    className="text-base font-bold"
+                  >
+                    {formatName(user)}
```

Comment:
<img width="578" alt="image" src="https://github.com/user-attachments/assets/0469d477-6ee7-42fe-87e6-9de49784379c" />

The tooltip works, however the long name is still displayed; issue isn't solved.

Reply:
Either, I can truncate or can decide some width or height

Reply:
> Either, I can truncate or can decide some width or height to that popover

Truncate the name on the card (Popover trigger). Popover is fine.

Reply:
done